runtime/synmenu.vim

```diff
@@ -18,6 +18,10 @@ fun! SetSyn(name)
   else
     let name = a:name
   endif
+  if name == "whitespace"
```

Comment:
Although it doesn't affect behavior, I guess using `a:name` here matches the code better?
```suggestion
  if a:name == "whitespace"
```